plasmapy_sphinx/__init__.py

```diff
@@ -70,8 +70,8 @@
 # This is the same check as the one at the top of setup.py
 import sys
 
-if sys.version_info < (3, 6):  # coverage: ignore
-    raise ImportError("plasmapy_sphinx does not support Python < 3.6")
+if sys.version_info < (3, 8):  # coverage: ignore
+    raise ImportError("plasmapy_sphinx does not support Python < 3.8")
 
 if sys.version_info >= (3, 8):
```

Comment:
Since the minimum version was just changed to 3.8, we should be able to remove this `if/else` block and drop the `importlib_metadata` line.